src/openrct2/actions/StaffHireNewAction.hpp

```diff
@@ -77,7 +77,10 @@ DEFINE_GAME_ACTION(StaffHireNewAction, GAME_COMMAND_HIRE_NEW_STAFF_MEMBER, Staff
     {
         GameAction::Serialise(stream);
 
-        stream << DS_TAG(_autoPosition) << DS_TAG(_staffType) << DS_TAG(_entertainerType) << DS_TAG(_staffOrders);
+        uint8_t casted_staffType = static_cast<uint8_t>(_staffType); //there's an error when directly casting in the DS_TAG() function for some reason
```

Comment:
You can't do this as you will break multiplayer. Staff type will no longer get assigned.

Reply:
Got it, thank you! I'll fix it soon. And sorry about opening and closing, I'm pretty new to this. I'll reopen and change to a draft.